src/main/java/uk/gov/hmcts/darts/arm/service/impl/DetsToArmBatchProcessResponseFilesImpl.java

```diff
@@ -134,33 +127,6 @@ private void updateOsrResponseReceivedAttributes(ObjectStateRecordEntity osr) {
         osrRepository.save(osr);
     }
 
-    private void deleteBlobDataAndEod(ExternalObjectDirectoryEntity armEod, ObjectStateRecordEntity osr) {
-        List<ExternalObjectDirectoryEntity> detsEods = externalObjectDirectoryRepository.findExternalObjectDirectoryByLocation(
```

Comment:
This was the only `findExternalObjectDirectoryByLocation()` usage, so can that method be eliminated, or will it see usage in the future async version of the flow?